src/bindings/python/src/pyopenvino/graph/op.hpp

```diff
@@ -6,6 +6,47 @@
 
 #include <pybind11/pybind11.h>
 
+#include "openvino/op/op.hpp"
+#include "pyopenvino/graph/discrete_type_info.hpp"
+
 namespace py = pybind11;
 
+/// Trampoline class to support inheritance from TorchDecoder in Python
+class PyOp : public ov::op::Op {
```

Comment:
Is it our convention to add `Py` to pyopenvino classes that are wrappers for C++ API classes?

Reply:
I'd say no, but I prefer to call these wrappers like this to show that they contain/work with some python objects.